pages/common/ps.md

```diff
@@ -13,3 +13,11 @@
 - Search for a process that matches a string:
 
 `ps aux | grep {{string}}`
+
+- List all processes of the current user in extra full format:
```

Comment:
What do you mean by "extra full format" here?

Reply:
The explanation of `-F - extra full format` I took from the `man` page.

It's the same as `--format user,pid,ppid,c,sz,rss,psr,stime,tty,time,cmd`

| format specifier| description |
| --- | --- |
| user | effective user name |
| pid | process ID |
| ppid | parent process ID |
| c | processor utilization |
| sz | size in physical pages of the core image of the process |
| rss | size of the non-swapped physical memory |
| psr | processor that process is currently assigned to |
| stime | time the command started |
| tty | the terminal |
| time | cumulative CPU time |
| cmd | command with all its arguments |

Reply:
Ah, ok. Sure, this works then 😺